andela/ah-sealteam-frontend
# #161278871 Users should be able to create, read, update and delete articles

#### What does this PR do?
```Creates CRUD functionality for articles ```
#### Description of Task to be completed?
```GET /articles/:slug```
```POST /articles```
```UPDATE /articles/:slug/update```
```For deletion click on the icon on  the read article page```
#### How should this be manually tested?
```
$ git fetch origin pull/19/head:feature/Users-should-be-able-to-create-read-update-and-delete-articles-161278871
$ git checkout feature/Users-should-be-able-to-create-read-update-and-delete-articles-161278871
$ source .env
$ npm install
$ npm start
```
#### What are the relevant pivotal tracker stories?

[#161278871](https://www.pivotaltracker.com/story/show/162694516)
 
#### Checklist:

- [ ] At least 4 people have reviewed my PR

- By @botnetdobbs and @gtsofa
